api-services/api.service.js

```diff
@@ -62,6 +62,17 @@ export class ApiService {
         return await this.actionRequest(requestConfig, responseOptions)
     }
 
+    /**
+     * Calls a `DELETE` request to an API.
+     *
+     * @param {Provide the request configuration} requestConfig
+     * @param {Provide responseOptions if needed} responseOptions
+     */
+    async delete (requestConfig, responseOptions) {
```

Comment:
Does `responseOptions` need a default to make it optional?

Reply:
At the moment, no. Where responseOptions gets checked, it shortcuts if null and just return the response